metrics/SystemMetrics.md

```diff
@@ -0,0 +1,35 @@
+
+## Supported platforms
+TODO: Add list of supported platforms.
+
+## Common metrics collection
+These are some default metrics supported by OpenCensus system metrics package/contrib. This is to ensure consistency across language implementations, as far as practical.
+
+| Metric Name                               | Unit  | Type                       | Description                                                                                                | Labels                 |
+|-------------------------------------------|-------|----------------------------|-----------------------------------------------------------------------------------------------|-----------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                                                          | Hostname, Mode          |
+| system/processes/created                  | 1     | Int64 Cumulative           | Total number of times a process was created.                                                      | Hostname                |
+| system/processes/running                  | 1     | Int64 Gauge                | Total number of running processes.                                                                | Hostname                |
+| system/disk/total                         | By    | Int64 Gauge                | The total available system disk space in bytes.                                                   | Hostname, Volume        |
+| system/disk/used                          | By    | Int64 Gauge                | The total used system disk space in bytes.                                                        | Hostname, Volume        |
+| process/heap/total                        | By    | Int64 Gauge                | The process's total allocated heap size in bytes.                                                 | Hostname                |
+| process/heap/used                         | By    | Int64 Gauge                | The process's total used heap size in bytes.                                                      | Hostname                |
+| process/memory/rss                        | By    | Int64 Gauge                | Resident memory size in bytes.                                                                    | Hostname                |
+| process/file_descriptor/max               | 1     | Int64 Gauge                | The maximum limit of file descriptor count.                                                       | Hostname                |
+| process/file_descriptor/open              | 1     | Int64 Gauge                | The number of open file descriptor count.                                                         | Hostname                |
+| process/start_time                        | s     | Int64 Cumulative           | The start time of the process since unix epoch in seconds.                                        | Hostname                |
```

Comment:
This has no meaning. As it will be constant. May be a  wall-clock time would be better.

Reply:
Good catch, removed. Thx